README.md

```diff
@@ -42,6 +44,14 @@ To complete the DevOps Academy, you will have to go through 8 classes with hands
 
 You will also be required to peer review at least 5 pull requests from other students to finish this course.
 
+There will be 2 modes of learning the content in these classes:
+
+1. Self-learning. Where you go through all the classes on your own pace and submit the exercises when you're ready.
+
+2. Face-to-Face Classes or Virtual Classes. Where the instructors will go through the content of each class and answer any questions or concerns you might have. To sign up for the classes you can email help@devopsacademy.com to get the presentation dates and the availability for the upcoming class. 
+
+**The first round of classes has been completed and there is currently no future classes planned. We are working on the plan of the new term and will update this README with those dates as soon as it becomes available. If you are interested in joining the upcoming classes, email us at help@devopsacademy.com.au.**
+
```

Comment:
We just opened the registrations for the next group, which will start on June 2nd.
Would need to update this text with that.. :)

Happy to approve once this is updated.

Reply:
done @kikobr82